runtime/doc/various.txt

```diff
@@ -454,10 +454,12 @@ g8			Print the hex values of the bytes used in the
 							*:verbose-cmd*
 When 'verbose' is non-zero, listing the value of a Vim option or a key map or
 an abbreviation or a user-defined function or a command or a highlight group
-or an autocommand will also display where it was last defined.  If it was
-defined manually then there will be no "Last set" message.  When it was
-defined while executing a function, user command or autocommand, the script in
-which it was defined is reported.
+or an autocommand will also display where it was last defined. If they were
+defined through lua api they will only be located if verbose level was
+greater then 0 when that map/abbrev/option/function.. was defined. So
```

Comment:
delete this line

Reply:
> If they were
defined through lua api they will only be located if verbose level was
greater then 0 when that map/abbrev/option/function.. was defined

This one ?

Reply:
no. only the line i commented on. I gave the suggested form above.